_config.yml

```diff
@@ -97,9 +100,14 @@ defaults:
   - scope:
       type: "health-care"
     values:
-      layout: "page"
+      layout: "page-breadcrumbs"
+      breadcrumb_1: "Health-Care"
```

Comment:
This will make the visible breadcrumb on the page "Health-Care". I think what we'd want is "Health Care" or "Health care" (to be consistent with what we've been doing on other parts of the site, it would be "Health Care".

In that case, we can omit the setting for page-breadcrumbs variable, as "Health Care" would be automatically set based on the collection name. Alternatively, this could be set to "Health Care" to achieve the same result.